### ISSUE TITLE
[UI] Move browser settings icon

### ISSUE THREAD

**Issue by timpulver** [quodlibet/quodlibet#1816]
<img width="651" alt="bildschirmfoto 2016-02-09 um 23 50 27" src="https://cloud.githubusercontent.com/assets/1055819/12933361/ef3c43f2-cf87-11e5-9cd1-6fb721200a41.png">

To change the filter list on the left, you currently have to click the settings icon on the right. It’s not clear that the settings belong to the filter pane on the left, it looks like they belong to the pane on the right. It would be more clear if the settings-icon was somewhere on the left.

**Comment by lazka** [comment]
The button belongs to the whole browser, the same is true for all browers.

**Comment by declension** [comment]
I agree that in some browsers this is a confusing place to find the options that you do (i.e. not just searching options).

For the Paned Browser (probably the highest priority here) we could add right-clickable column headings which would be somewhat consistent with the idea of right clicking column headers in the songlist to configure _those_ columns...

**Comment by zsau** [comment]
If that context menu also allowed selecting a saved pattern, it could go a long way towards making the Paned Browser more fluid (ideally along with #1256).
